modules/ocf_mirrors/manifests/projects/openbsd.pp

```diff
@@ -0,0 +1,25 @@
+class ocf_mirrors::projects::openbsd {
+  file {
+    '/opt/mirrors/project/openbsd':
+      ensure  => directory,
+      source  => 'puppet:///modules/ocf_mirrors/project/openbsd',
+      owner   => mirrors,
+      group   => mirrors,
+      mode    => '0755',
+      recurse => true;
+  }
+
+  ocf_mirrors::monitoring { 'openbsd':
+    type          => 'unix_timestamp',
+    upstream_host => 'mirrors.mit.edu',
```

Comment:
Likewise here, I think this should be `https://ftp.openbsd.org/pub/OpenBSD/timestamp` instead so we're checking the ultimate source of packages, not another mirror that could be outdated.